packages/tpetra/core/src/Tpetra_Details_DistributorActor.cpp

```diff
@@ -42,14 +42,17 @@
 namespace Tpetra {
 namespace Details {
 
-  DistributorActor::DistributorActor() {
+  DistributorActor::DistributorActor()
+    : mpiTag_(1)
```

Comment:
I think the choice of `1` here is arbitrary, if so we could toss in a comment saying so

Reply:
@cwpearson I'm strongly opposed to comments unless they're absolutely necessary, I'd much prefer to express myself clearly in code.  What if instead we added `mpiTagDefault = 1` in an anonymous namespace, then use that variable to initialize `DistributorActor::mpiTag_`?  Would that make it clear enough to you that this value was arbitrary?